scipy/stats/morestats.py

```diff
@@ -2150,7 +2192,27 @@ def ansari(x, y):
     With a p-value of 0.00628, the test provides strong evidence that
     the scales of the distributions from which the samples were drawn
     are not equal.
+
+    We can use the ``alternative`` parameter to perform one-tailed test.
+    In the above example, the scale of `x1` is greater than `x3` and so
+    the ratio of scales of `x1` and `x3` is greater than 1. This means
+    that the p-value when ``alternative='greater'`` should be near 0 and
+    hence we should be able to reject the null hypothesis:
+
+    >>> ansari(x1, x3, alternative='greater')
+    AnsariResult(statistic=452.0, pvalue=0.0031401394)
+
+    As we can see, the p-value is indeed less than the level of significance.
```

Comment:
If we want to make this statement, we would need to define, before performing the test, what we will consider to be significant.

Reply:
I was thinking the same. Getting rid of the level of significance altogether will be better and simpler. So, I will rephrase that statement.